src/Router.cpp

```diff
@@ -17,12 +17,17 @@ void Router::routeRequest(const HTTPRequest &request, HTTPResponse &response)
 		CGIHandler cgiHandler;
 		cgiHandler.setFDsRef(_FDsRef);
 		cgiHandler.setPollFd(_pollFd);
-		// cgiHandler.handleRequest(request, response);
 		cgiHandler.handleRequest(request, response);
-		// std::cout << std::endl << std::endl << std::endl << std::endl;
-		// std::cout << response;
+		return;
 	}
-	else // it is a static request
+	else if (request.getMethod() == "POST")
+	{
+		std::cout << "Router: POST request" << std::endl;
+
+		UploadHandler uploadHandler;
+		uploadHandler.handleRequest(request, response);
+	}
+	else
```

Comment:
Can you just check if there is POST request that would not be CGI nor file upload? And eventually add a dummy else if condition (since we are already here reading this), or eventually open a linear issue. 🙏

Reply:
I modified the code so uploadHandler.handleRequest(request, response); is only call for an upload (not another kind of POST) , and I added 🥗
I'll research the matter